packages/coinstac-api-server/test/test-setup.js

```diff
@@ -100,6 +100,35 @@ helperFunctions.getRethinkConnection()
       },
     ],
   },
+  {
+    id: 'test-pipeline-local',
+    name: 'Local Pipeline',
+    description: 'Local Test description',
+    owningConsortium: 'test-cons-2',
+    shared: true,
+    steps: [
+      {
+        id: 'UIKDl-local1',
+        controller: { type: 'local' },
+        computations: [
+          compInsertResult.changes[0].new_val.id,
+        ],
+        inputMap: {
+          start: { value: 1 },
+        },
+      },
+      {
+        id: 'UIKDl-local2',
+        controller: { type: 'local' },
+        computations: [
+          compInsertResult.changes[0].new_val.id,
+        ],
+        inputMap: {
+          start: { fromCache: { step: 0, variable: 'sum', label: 'Sum' } },
```

Comment:
While this works, I'd prefer to reference the compspec for the Label attribute

Reply:
That's fine. I'll take it out of the test file here and ensure it's looking to Computation on the branch I'm working on now.